index.ts

```diff
@@ -130,146 +136,155 @@ if (syncRepoLabels) {
-                            gitHubSource.reactOnComment(sourceComment.id, 'rocket')
-                            console.info('Successfully created new comment on issue')
-                        })
-                    } else {
-                        // edited or deleted
-                        gitHubTarget.getComments(targetIssueNumber).then(targetComments => {
-                            const targetCommentMatch = utils.findTargetComment(sourceComment, targetComments)
-                            if (targetCommentMatch) {
-                                if (action == 'edited') {
-                                    gitHubTarget.editComment(targetCommentMatch.id, issueCommentBody).then(response => {
-                                        // set target comment id for GH output
-                                        core.setOutput('comment_id_target', response.data.id)
-                                        console.info('Successfully updated a comment on issue')
-                                    })
-                                } else if (action == 'deleted') {
-                                    gitHubTarget.deleteComment(targetCommentMatch.id)
-                                }
-                            }
-                        })
+            if (action == 'created') {
+                gitHubTarget.createComment(targetIssueNumber, issueCommentBody).then(response => {
+                    // set target comment id for GH output
+                    core.setOutput('comment_id_target', response.data.id)
+                    gitHubSource.reactOnComment(sourceComment.id, 'rocket')
+                    console.info('Successfully created new comment on issue')
+                })
+            } else {
+                // edited or deleted
+                gitHubTarget.getComments(targetIssueNumber).then(targetComments => {
+                    const targetCommentMatch = utils.findTargetComment(sourceComment, targetComments)
+                    if (targetCommentMatch) {
+                        if (action == 'edited') {
+                            gitHubTarget.editComment(targetCommentMatch.id, issueCommentBody).then(response => {
+                                // set target comment id for GH output
+                                core.setOutput('comment_id_target', response.data.id)
+                                console.info('Successfully updated a comment on issue')
+                            })
+                        } else if (action == 'deleted') {
+                            gitHubTarget.deleteComment(targetCommentMatch.id)
+                        }
                     }
                 })
+            }
+        })
+        break
+    case 'issues':
+        // If the issue was updated, we need to sync labels
+        if (skipSync) break
+        const issueBody = utils.getIssueTargetBody(issue)
+
+        switch (action) {
+            case 'opened':
```

Comment:
we talked briefly on Tuesday about it already and I'd drop the `opened` case.
Reason being is that when a user opens an issue and labels it on creation then 2 copies are created in the target as they're running in concurrency for opened and labeled event and at that point no target issue exists yet.

I didn't use the opened trigger prior in my testing as I've always only regarded the `labeled` event (same for product-hub).